draft-ietf-spice-sd-cwt.md

```diff
@@ -399,7 +433,7 @@ Multiple levels of tags in a key are not permitted.
 
 SD-CWT is modeled after SD-JWT, with adjustments to align with conventions in CBOR and COSE. An SD-CWT MUST include the protected header parameter `typ` {{!RFC9596}} with the value "application/sd-cwt" in the SD-CWT.
 
-An SD-CWT is a CWT containing the "blinded claim hash" of at least one blinded claim in the CWT payload.
+An SD-CWT is a CWT containing the "blinded claim hash" of zero or more blinded claim in the CWT payload.
```

Comment:
Is this true? wouldn't this yield:

```
"_sd": [],
```

which is a commitment to zero disclosures at this level of the map?

Reply:
yes

Reply:
An SD-CWT does not need to have any disclosable claims present in its payload, or any arrays or maps inside the payload.

When an SD-CWT has no disclosable claims, the redacted map key and redacted array element MUST NOT be present anywhere in the payload.

Reply:
Move this maybe to a better spot?

Reply:
I didn't move this, but I think it is much nicer now as of https://github.com/ietf-wg-spice/draft-ietf-spice-sd-cwt/pull/36/commits/b7eb6904a8e71320a7a2d15b944a5c0670c9bdb8